src/python/grpcio/grpc/_cython/_cygrpc/grpc.pxi

```diff
@@ -21,6 +21,7 @@ ctypedef unsigned char uint8_t
 ctypedef int int32_t
 ctypedef unsigned uint32_t
 ctypedef long int64_t
+ctypedef ssize_t intptr_t
```

Comment:
Any reason we are defining these here? Please consider replacing this typedef (and the above) with `from libc.stdint cimport intptr_t` instead of redefining here if nothing breaks. [`libc.stdint` in Cython](https://github.com/cython/cython/blob/master/Cython/Includes/libc/stdint.pxd) already has these definitions.